cmd/plugin/cli/install.go

```diff
@@ -50,22 +50,24 @@ func InstallCmd() *cobra.Command {
 	cmd.Flags().String(installer.StosVersionFlag, "", "version of storageos operator")
 	cmd.Flags().String(installer.StosOperatorYamlFlag, "", "storageos-operator.yaml path or url")
 	cmd.Flags().String(installer.StosClusterYamlFlag, "", "storageos-cluster.yaml path or url")
+	cmd.Flags().String(installer.StosPortalConfigYamlFlag, "", "storageos-portal-configmap.yaml path or url")
 	cmd.Flags().String(installer.EtcdClusterYamlFlag, "", "etcd-cluster.yaml path or url")
 	cmd.Flags().String(installer.EtcdOperatorYamlFlag, "", "etcd-operator.yaml path or url")
 	cmd.Flags().Bool(installer.IncludeEtcdFlag, false, "install non-production etcd from github.com/storageos/etcd-cluster-operator")
 	cmd.Flags().Bool(installer.EtcdTLSEnabledFlag, false, "etcd cluster is tls enabled")
 	cmd.Flags().Bool(installer.SkipEtcdEndpointsValFlag, false, "skip validation of etcd endpoints")
 	cmd.Flags().Bool(installer.SkipStosClusterFlag, false, "skip storageos cluster installation")
+	cmd.Flags().Bool(installer.EnablePortalFlag, false, "enable storageos portal during installation")
 	cmd.Flags().String(installer.EtcdEndpointsFlag, "", "endpoints of pre-existing etcd backend for storageos (implies not --include-etcd)")
 	cmd.Flags().String(installer.EtcdSecretNameFlag, consts.EtcdSecretName, "name of etcd secret in storageos cluster namespace")
 	cmd.Flags().String(installer.StosConfigPathFlag, "", "path to look for kubectl-storageos-config.yaml")
 	cmd.Flags().String(installer.EtcdNamespaceFlag, consts.EtcdOperatorNamespace, "namespace of etcd operator and cluster to be installed")
 	cmd.Flags().String(installer.StosOperatorNSFlag, consts.NewOperatorNamespace, "namespace of storageos operator to be installed")
 	cmd.Flags().String(installer.StosClusterNSFlag, consts.NewOperatorNamespace, "namespace of storageos cluster to be installed")
 	cmd.Flags().String(installer.EtcdStorageClassFlag, "", "name of storage class to be used by etcd cluster")
-	cmd.Flags().String(installer.AdminUsernameFlag, "", "storageos admin username (plaintext)")
-	cmd.Flags().String(installer.AdminPasswordFlag, "", "storageos admin password (plaintext)")
-	cmd.Flags().String(installer.PortalKeyPathFlag, "", "path to portal manager key")
+	cmd.Flags().String(installer.AdminUsernameFlag, "", "storageos admin username, also used as the portal clientID (plaintext)")
```

Comment:
I don't suggest to add `, also ...` to the flag, because this is a temporary solution and we will forget to come back here and fix text :)